### DISCUSSION TITLE
购买专业版后无法授权iPhone

### DISCUSSION THREAD

**Discussion by fate126126** [getlantern/lantern-forum-cn#240] [General]
购买专业版后无法授权iPhone      18217316857@139.com

**Comment by MaxMKMax** [comment]
@fate126126 谢谢您的反馈. 好的明白了! 我帮您和团队说明这个问题。我帮你紧急解决问题。我现在马上通知技术团队!

**Reply by MaxMKMax** [reply]
@fate126126 您好。当前版本仍然存在登录问题。请给我们几天的时间来解决，我们会尽快更新您的信息。我们将很快发布最新版本。抱歉让您久等了。稍后我们还将补偿您更长的专业版。

**Comment by good9flower** [comment]
我也遇到这个情况，当前用的版本：7.5.1 、构建版本14 ，SDK 7.6.14 。我的账号good9flower@gmail.com,请尽快解决，下个月我的账号要到期了。

**Reply by good9flower** [reply]
我“当前用的版本：7.5.1 、构建版本14 ，SDK 7.6.14”指的是Iphone上的蓝灯版本号。

**Reply by MaxMKMax** [reply]
@good9flower 嗯嗯. 谢谢您的反馈! 好的. 我会帮您询问看有什么解决方案. 非常抱歉影响了您使用蓝灯!
